github/ci/prow-deploy/files/jobs/kubevirt/community/community-postsubmits.yaml

```diff
@@ -18,7 +18,7 @@ postsubmits:
       decorate: true
       spec:
         containers:
-          - image: kubevirt/builder@sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab
+          - image: quay.io/kubevirt/builder@sha256:0e2dde4051711e243dbb7be57a7c10f67a3b33bc764304e14297e324deaddee0
```

Comment:
Why not using the same sah256 that we had in docker.io? It is available on quay too

Reply:
i dont think it is ?
```
project-infra (map) $ docker pull quay.io/kubevirt/builder@sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab
Error response from daemon: manifest for quay.io/kubevirt/builder@sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab not found: manifest unknown: manifest unknown
```

Reply:
hmm not sure what's going on, after `docker system prune -a` I'm still able to pull that concrete digest:
```
$ docker pull quay.io/kubevirt/builder@sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab
sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab: Pulling from kubevirt/builder
b0fe058ffda2: Pull complete 
dfe6ecd6d77b: Pull complete 
48699dca5da3: Pull complete 
b1833078a99d: Pull complete 
d1de41b0fa4e: Pull complete 
5cf469971bf6: Pull complete 
95060617e71b: Pull complete 
aafcb618b9d8: Pull complete 
70691977293d: Pull complete 
d0f8d8b071eb: Pull complete 
Digest: sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab
Status: Downloaded newer image for quay.io/kubevirt/builder@sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab
quay.io/kubevirt/builder@sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab
```

Reply:
if you docker logout ?
maybe since you have creds

it doesnt appear here
https://quay.io/repository/kubevirt/builder?tag=latest&tb=tags

and i would go for the safe side, 
i also don't see why its very important to keep the previous repo hash,
we would change it anyhow on the next image update

Reply:
> if you docker logout ?
> maybe since you have creds

sure, works too:
```
$ docker logout
Removing login credentials for https://index.docker.io/v1/
13:41:40 fgimenez@localhost:~/workspace/redhat/kubevirt$ docker system prune
WARNING! This will remove:
  - all stopped containers
  - all networks not used by at least one container
  - all dangling images
  - all dangling build cache

Are you sure you want to continue? [y/N] y
Deleted Images:
untagged: quay.io/kubevirt/builder@sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab
deleted: sha256:c88a484b960b86d5b88a8258df50914417b1dbcc4e67ae61a666216e6bc12a75
deleted: sha256:fb53840830bf2cc066a3745c9dc9ecd95a2c1330fd32440cd2e89e8613cd86ba
deleted: sha256:163354003d294ce41ba50871f17d618e0a2a26f8d29dd19b7d3f7d3ef5c5d647
deleted: sha256:825a577f6da48e26ece631bc97bf57062e55ec529587596424f1548f25874d88
deleted: sha256:f74aaf069871c397fdd405bc255775fea05427d82f8bf5b1b2a807f30553abbc
deleted: sha256:b4b466b066b7f1e3541334d98ca6173ebc9505789d5aea282c411a9dfbe5e528
deleted: sha256:c5f4f951d83841b31ec377818ecdc9d4c8af31fced6964f2ff8e310c600aae60
deleted: sha256:e900de34869b90d587536da6535efa303f368a8d880d03c2f4f9eac10d2742a2
deleted: sha256:0089a7ba1838a8dbad87408067fdaeaae644dbf4138528b7ca184efa8c2db2ac
deleted: sha256:6fe74a8ad19c69157a2e0ea958d268728c4dc25234ccff0be71a075ef99fe126
deleted: sha256:a66ff0d17109a09a6041858f84a85ba8df13100008b2ade55779af3198cd7b00

Total reclaimed space: 1.999GB
13:41:48 fgimenez@localhost:~/workspace/redhat/kubevirt$ docker pull quay.io/kubevirt/builder@sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab
sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab: Pulling from kubevirt/builder
b0fe058ffda2: Pull complete 
dfe6ecd6d77b: Pull complete 
48699dca5da3: Pull complete 
b1833078a99d: Pull complete 
d1de41b0fa4e: Pull complete 
5cf469971bf6: Pull complete 
95060617e71b: Pull complete 
aafcb618b9d8: Pull complete 
70691977293d: Pull complete 
d0f8d8b071eb: Pull complete 
Digest: sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab
Status: Downloaded newer image for quay.io/kubevirt/builder@sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab
quay.io/kubevirt/builder@sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab
```

> it doesnt appear here
> https://quay.io/repository/kubevirt/builder?tag=latest&tb=tags

yep, it should not appear under the tags tab, we are referring to it by digest

> i also don't see why its very important to keep the previous repo hash,

I think we should make sure that we are using the exact same image, we can try to trigger the job using the original digest but pointing to quay, wdyt?

Reply:
> I think we should make sure that we are using the exact same image, we can try to trigger the job using the original digest but pointing to quay, wdyt?

I verified it, as follows:
downloaded both from quay and docker, saw both have the same local `IMAGE ID`,
docker inspect that image and saw it has the two labels of quay and docker.io (with the different hashes)

IMO we should not worry, let me know what you think and i will do it,
i wonder if it will even work on CI itself

Reply:
maybe you need `docker logout quay.io` ?
didn't it logout just from docker.io according printed?
```
$ docker logout
Removing login credentials for https://index.docker.io/v1/
```

Reply:
good catch, but login out of quay works too, the image is public:
```
14:15:58 fgimenez@localhost:~/workspace/redhat/kubevirt$ docker logout quay.io
Removing login credentials for quay.io
14:17:38 fgimenez@localhost:~/workspace/redhat/kubevirt$ docker image ls
REPOSITORY                       TAG                 IMAGE ID            CREATED             SIZE
rpardini/docker-registry-proxy   0.6.2               6bbe4e47a504        2 months ago        12.3MB
14:17:46 fgimenez@localhost:~/workspace/redhat/kubevirt$ docker pull quay.io/kubevirt/builder@sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab
sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab: Pulling from kubevirt/builder
b0fe058ffda2: Pull complete 
dfe6ecd6d77b: Pull complete 
48699dca5da3: Pull complete 
b1833078a99d: Pull complete 
d1de41b0fa4e: Pull complete 
5cf469971bf6: Pull complete 
95060617e71b: Pull complete 
aafcb618b9d8: Pull complete 
70691977293d: Pull complete 
d0f8d8b071eb: Pull complete 
Digest: sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab
Status: Downloaded newer image for quay.io/kubevirt/builder@sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab
quay.io/kubevirt/builder@sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab
```
Let's verify it with the same digest on CI, I don't see any reason for it to fail.

Reply:
lets run this as sanity please, since this PR have many,  i cant rehearse one specific

if it will work i will do it for the other commits as well (unless you say its not needed)
https://github.com/kubevirt/project-infra/pull/1105

relevant commits with the same phenomena (if you want we can create a sanity check for each of them)
ff9c1d7 infra, Update indexpagecreator to point to quay.io
e0b759d infra, Update kubevirtci/release-blocker related
ed8bc79 infra, Update kubevirt/builder images to point to quay

Reply:
Ok I finally managed to get the same error by turning off the local docker proxy, all good now, we can continue with the digests as they are in the PR, sorry for all the noise :)